ui/apps/platform/package.json

```diff
@@ -121,8 +118,6 @@
     },
     "devDependencies": {
         "@babel/core": "7.12.3",
-        "@cypress/code-coverage": "^3.9.11",
-        "@cypress/instrument-cra": "^1.4.0",
```

Comment:
* `"@jest/reporters@^26.6.2"` still has `istanbul-lib-coverage "^3.0.0"` under its dependencies.
* `"@cypress/code-coverage@^3.9.11"` had `nyc "15.1.0"` under its dependencies, so it seems redundant that it was added explicitly. I will verify that jest code coverage still works without it.

By the way, `babel-plugin-istanbul@6.0.0, babel-plugin-istanbul@^6.0.0` has `"@istanbuljs/load-nyc-config" "^1.0.0"` under its dependencies, and Jest still refers to the Babel plugin.

Reply:
Okay, deleted both because it was redundant to include them as explicit devDependencies.

Also deleted 2 references to coverage and 1 leftover reference to demo in ui/Makefile (good eyes, Van)